Zappy_IA_Report/DTC/Video/Tutorial2.py

```
@@ -0,0 +1,29 @@
+df = df.drop(columns=['PLAYER', 'CS'])
+
+print("\ndf.info: ", df.info())
+print("\ndf.describe: ", df.describe().round(3))
+X1 = df.iloc[:, 0:14]
+X2 = df.iloc[:, 0:14]
+
+from sklearn.preprocessing import StandardScaler
+
+scaleStandard = StandardScaler()
+X1 = scaleStandard.fit_transform(X1)
+X1 = pd.DataFrame(X1, columns=['YRS' ,'G' ,'AB' ,'R' ,'H' ,'2B' ,'3B' ,'HR' ,'RBI' ,'BB','SO' ,'SB' ,'BA', 'HOF'])
```

Flake8 (7.2.0)

[error] whitespace before ','

(E203)